include/picongpu/plugins/radiation/Radiation.hpp

```diff
@@ -282,9 +282,12 @@ namespace picongpu
                     // write backup file
                     if(isMaster)
                     {
-                        writeOpenPMDfile(
-                            tmp_result,
-                            restartDirectory + "/" + speciesName + std::string("_radRestart"));
+                        /*
+                         * No need to keep the Series open for checkpointing, so
+                         * just quickly open it here.
+                         */
+                        std::optional<::openPMD::Series> tmp;
```

Comment:
See here @psychocoderHPC 
This is the Series for checkpointing, just quickly opened here and then closed again